modules/go-deleting-argocd-instance.adoc

```diff
@@ -14,19 +14,19 @@ Delete the Argo CD instances added to the namespace of the GitOps Operator.
 
 [source,terminal]
 ----
-$ oc delete gitopsservice cluster -n openshift-gitops
+$ oc delete gitopsservice cluster
 ----
 
 [NOTE]
 ====
 You cannot delete an Argo CD cluster from the web console UI.
 ====
 
-After the command runs successfully all the Argo CD instances will be deleted from the `openshift-gitops` namespace.
+After the above command runs successfully the default GitOps Service and its respective Argo CD instance will be deleted from the `openshift-gitops` namespace.
 
-Delete any other Argo CD instances from other namespaces using the same command:
+Delete any other custom ArgoCD instances from other namespaces using the following command:
```

Comment:
Slight revision: 

Delete any other custom ArgoCD instances from other namespaces by entering the following command: 

All commands should be introduced with "by entering" or "by running" (see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output if you're interested).